charts/opentelemetry-operator/values.yaml

```diff
@@ -198,6 +198,7 @@ kubeRBACProxy:
 ## They also enable the sidecar injection for OpenTelemetryCollector and Instrumentation CR's
 admissionWebhooks:
   create: true
+  enablePodOperationsWebhook: true
```

Comment:
we should move this under the `pods` key below and just call it `enabled`. We should also add a comment to inform users that doing this will break sidecar injection and instrumentation